### PR TITLE
docs 3rd party tools: update java wrapper, add cmd tool to add article

### PR DESCRIPTION
**Pull request by Strubbl**
| Q             | A
| ------------- | ---
| Bug fix?      | 
| New feature?  | 
| BC breaks?    | 
| Deprecations? | 
| Tests pass?   | 
| Documentation | yes
| Translation   | yes
| Fixed tickets | 
| License       | MIT

### CODE REVIEW

**Comment by Strubbl** [comment]
French translation needs some help here

File: docs/en/developer/api.rst [hunk 1]
```diff
@@ -263,8 +263,9 @@ Third party resources
...
+- `Java wrapper for the wallabag API <https://github.com/di72nn/wallabag-api-wrapper>`_ by Dmitriy Bogdanov.
 - `.NET library for the wallabag v2 API <https://github.com/jlnostr/wallabag-api>`_ by Julian Oster.
 - `Python API for wallabag <https://github.com/foxmask/wallabag_api>`_ by FoxMaSk, for his project `Trigger Happy <https://blog.trigger-happy.eu/>`_.
 - `A plugin <https://github.com/joshp23/ttrss-to-wallabag-v2>`_ designed for `Tiny Tiny RSS <https://tt-rss.org/gitlab/fox/tt-rss/wikis/home>`_ that makes use of the wallabag v2 API. By Josh Panter.
-- `Golang wrapper for the wallabag API <https://github.com/Strubbl/wallabago>`_ by Strubbl, for his project `wallabag-stats graph <https://github.com/Strubbl/wallabag-stats>`_.
+- `Golang wrapper for the wallabag API <https://github.com/Strubbl/wallabago>`_ by Strubbl, for his projects `wallabag-stats graph <https://github.com/Strubbl/wallabag-stats>`_ and the command line tool `wallabag-add-article <https://github.com/Strubbl/wallabag-add-article>`_.
+- Tool to automatically download Wallabag articles into your local computer or Kobo ebook reader `wallabako <https://gitlab.com/anarcat/wallabako>`_ by Antoine Beaupré.
```
Comment (tcitworld) [inline docs/en/developer/api.rst]: Maybe use @anarcat instead.

Comment (Strubbl) [inline docs/en/developer/api.rst]: Yes maybe. Some have nick names, some their real names. As he posts his name to his github and gitlab profiles, i thought i would prefer name. I don't care, but maybe @anarcat wants to decide.

Comment (anarcat) [inline docs/en/developer/api.rst]: please use @anarcat

**Comment by nicosomb** [comment]
@Strubbl: we can merge when you'll add @anarcat username 👍

**Comment by nicosomb** [comment]
Thank you @Strubbl

**Pull request merged**